front/app/containers/Admin/index.tsx

```diff
@@ -141,26 +161,30 @@ const AdminPage = memo<Props & WithRouterProps>(
       pathname.includes('admin/dashboard') ||
       pathname.includes('admin/processing') ||
       pathname.includes('admin/insights');
+
     const whiteBg =
       endsWith(pathname, 'admin/moderation') ||
       pathname.includes('admin/dashboard') ||
-      pathname.includes('admin/processing');
+      pathname.includes('admin/processing') ||
+      pathname.includes('admin/insights/reports');
```

Comment:
Ideally, but this is OS leftover work and not blocking nor pressing, the idea whould be for a module to fully define itself, and this is a (low impact) reference to a module from the core app which should be avoided, since there's no mechanism in place to achive that, I'm not sure we want invest to effort of fixing now, but it there would be a low effort solution to fix that's be great !  Maybe module config could include their white bg and fullwidth tabs and they could be dynamically added here ?

Reply:
Hey, thanks for this! I couldn't find an easy fix for this? Should we note it somewhere and address down the line?

Reply:
Yes, you can log in the i1 epic, there's something I'd like to try to fix this